qemu/tests/balloon_check.py

```diff
@@ -24,12 +24,14 @@ def __init__(self, test, params, env):
 
         self.vm = env.get_vm(params["main_vm"])
         self.session = self.get_session(self.vm)
+        self.env["memory_stable"] = False
         if self.params.get('os_type') == 'windows':
             sleep_time = 180
         else:
             sleep_time = 90
         logging.info("Waiting %d seconds for guest's applications up" % sleep_time)
         time.sleep(sleep_time)
+        self.env["memory_stable"] = True
```

Comment:
I think the name `"balloon_test_setup_ready"` is better.

Reply:
Modified,thanks